include/swift/AST/DiagnosticsSema.def

```diff
@@ -2415,6 +2415,9 @@ NOTE(multiple_override_prev,none,
 ERROR(override_unavailable, none,
       "cannot override %0 which has been marked unavailable%select{|: %1}1",
       (DeclBaseName, StringRef))
+NOTE(suggest_removing_override, none,
+     "remove 'override' modifier to declare a new %0",
+     (DeclBaseName))
```

Comment:
Not a hard requirement, but it would be great to put this in YAML diagnostic file as well to help with the ongoing localisation efforts!